src/rust/grapl-config/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 
 [dependencies]
 log = "0.4.*"
-sqs-lambda = "0.20.*"
+sqs-lambda = "0.21.0"
```

Comment:
Any particular reason we set patch number for this but not others?

Same for graph-generator-lib